plugins/parquet/parquetembed.cpp

```diff
@@ -162,10 +163,26 @@ arrow::Status ParquetHelper::openWriteFile()
         writeOptions.filesystem = filesystem;
         writeOptions.base_dir = destination;
         writeOptions.partitioning = partitioning;
-        writeOptions.existing_data_behavior = arrow::dataset::ExistingDataBehavior::kOverwriteOrIgnore;
+        writeOptions.existing_data_behavior = overwrite ? arrow::dataset::ExistingDataBehavior::kOverwriteOrIgnore : arrow::dataset::ExistingDataBehavior::kError;
     }
     else
     {
+        StringBuffer filename;
+        StringBuffer path;
+        splitFilename(destination.c_str(), nullptr, &path, &filename, nullptr, false);
+        Owned<IDirectoryIterator> itr = createDirectoryIterator(path.str(), filename.append("*.parquet").str());
```

Comment:
minor: not likely, but if there was a matching directory, wouldn't want to pick up, and can avoid by passing includedirs = false here.

Reply:
Changed includedirs to false.